buildSrc/src/main/groovy/Classpaths.groovy

```diff
@@ -287,4 +287,31 @@ class Classpaths {
         Configuration config = p.configurations.getByName(configName)
         addDependency(config, GUAVA_GROUP, GUAVA_NAME, GUAVA_VERSION)
     }
+
+    static void inheritParquetHadoop(Project p, String configName = JavaPlugin.IMPLEMENTATION_CONFIGURATION_NAME) {
+        Configuration config = p.configurations.getByName(configName)
+        addDependency(config, 'org.apache.parquet', 'parquet-hadoop', '1.13.0')
+    }
+
+    /** configName controls only the Configuration's classpath, all transitive dependencies are runtimeOnly */
+    static void inheritParquetHadoopConfiguration(Project p, String configName = JavaPlugin.IMPLEMENTATION_CONFIGURATION_NAME) {
+        Configuration config = p.configurations.getByName(configName)
+        addDependency(config, 'org.apache.hadoop', 'hadoop-common', '3.3.3') {
+            it.because('Required for org.apache.hadoop.conf.Configuration')
+            it.setTransitive(false)
+            // Do not take any extra dependencies of this project transitively. We can add any required additional
+            // dependencies separately, as done for woodstox, shaded-guava, etc. below.
```

Comment:
The deleted version of this commentary was much more verbose. Are we losing valuable context?

Reply:
It felt like some of the commentary was a bit repetitive and incomplete to me and that's why I summarized it. 
For example, in 
```
        // do not take any dependencies of this project,
        // we just want a few classes (Configuration, Path) for
        // simplified prototyping work, and api compatibility.
```
we use a number of classes for a number of reasons. So felt incomplete to keep just Configuration and Path.
Similarly, in 
```
        transitive = false
        // if we actually need any more of hadoop at runtime,
        // we can add more jars w/ transitive=false,
        // or replace transitive=false here w/ more exclusions;
        // (we want to avoid pulling in netty, loggers, jetty-util, guice and asm).
```
I felt I summarized this in fewer words. ParquetHadoop has lots and lots of dependencies and we are avoiding all of them. So felt incomplete to name a few.

But I can definitely add it back if you think that the original comments adds more context.